src/audio/google/google_rtc_audio_processing.c

```diff
@@ -518,6 +518,22 @@ static int google_rtc_audio_processing_free(struct processing_module *mod)
 	return 0;
 }
 
+static bool is_ref_buffer(struct comp_dev *dev, struct comp_buffer *b)
+{
+#if CONFIG_IPC_MAJOR_4
+	return IPC4_SINK_QUEUE_ID(buf_get_id(b)) == SOF_AEC_FEEDBACK_QUEUE_ID;
+#else
+	return b->source->pipeline->pipeline_id != dev->pipeline->pipeline_id;
+#endif
+}
+
+static enum sof_ipc_frame bits_fmt(int bits)
+{
+	if (bits == 32)
+		return SOF_IPC_FRAME_S32_LE;
```

Comment:
this seems weird and not scalable if we introduce floats natively into SOF

Reply:
Agreed, I'll go with a runtime switch and remove the kconfig